tesseract/tesseract_planning/tesseract_process_planners/CMakeLists.txt

```diff
@@ -1,13 +1,14 @@
 cmake_minimum_required(VERSION 3.5.0)
 project(tesseract_process_planners VERSION 0.1.0 LANGUAGES CXX)
 
-find_package(tesseract REQUIRED)
+find_package(tesseract_common REQUIRED)
+find_package(tesseract_motion_planners REQUIRED)
 find_package(Eigen3 REQUIRED)
 
 list(FIND CMAKE_CXX_COMPILE_FEATURES cxx_std_11 CXX_FEATURE_FOUND)
 
 add_library(${PROJECT_NAME} SHARED src/${PROJECT_NAME}/process_definition.cpp)
-target_link_libraries(${PROJECT_NAME} PUBLIC tesseract::tesseract)
+target_link_libraries(${PROJECT_NAME} PUBLIC tesseract::tesseract_common tesseract::tesseract_motion_planners_core)
```

Comment:
I'm wondering why it built for me without this change

Reply:
I'm building this in kinetic

Reply:
You were including tesseract package, which includes all most all of the libraries. I just reduced it to include only the libraries it was using.